README.adoc

```diff
@@ -22,7 +22,7 @@ Mutual TLS means that connecting clients need to have a client certificate befor
 |Client certificates will expire, and hence need to recreate these for all clients periodically, which introduces a lot of overhead and potential for errors
 
 |
-|Have to maintain a CA for signing client certs
+|Have to maintain a CA for signing client certs. (Have you thought about how you will revoke access should this need arise?)
```

Comment:
This is possibly using CRLs (Certificate Revocation Lists). How we manage the updating and distribution of these lists is left as an exercise for the reader.